passport_strategies/initLocalAuth.js

```diff
@@ -13,11 +13,6 @@ module.exports = function localAuthStrategy(passport) {
           message: 'Get REKT son',
         });
       }
-
```

Comment:
Don't forget to validate password. This was a placeholder for us since we only do email validation right now